Source/WebCore/dom/Element.cpp

```diff
@@ -1583,6 +1583,23 @@ int Element::clientHeight()
     return 0;
 }
 
+double Element::currentCSSZoom()
+{
+    protectedDocument()->updateStyleIfNeeded();
+
+    auto initialZoom = [&]() -> float {
+        Ref resolver = protectedDocument()->styleScope().resolver();
+        const Document& document = resolver->document();
```

Comment:
I don't think we normally const local variables. You should also use `auto&` or maybe even `Ref`.